grease/src/Grease/FunctionOverride.hs

```diff
@@ -19,42 +19,51 @@ module Grease.FunctionOverride
 import Control.Lens ((^.), to)
 import Data.Bool (Bool(..))
 import Data.Maybe (Maybe(..), mapMaybe)
-import qualified Data.Vector as Vec
+import Data.Vector qualified as Vec
 import Prelude (($), (.), otherwise, toInteger)
 import System.IO (IO)
 
 import Control.Applicative (pure)
 import Control.Monad.State (MonadState(..), StateT(..), evalStateT)
 import Control.Monad.IO.Class (liftIO)
-import qualified Data.ByteString as BS
-import qualified Data.ByteString.Char8 as BSC
+import Data.ByteString qualified as BS
+import Data.ByteString.Char8 qualified as BSC
 import Data.Either (Either(Left, Right))
 import Data.Functor ((<$>), fmap)
 import Data.Functor.Identity (Identity(Identity, runIdentity))
-import qualified Data.List as List
-import qualified Data.Map as Map
-import qualified Data.Sequence as Seq
+import Data.List qualified as List
+import Data.Map qualified as Map
+import Data.Sequence qualified as Seq
 import Data.Set (Set)
-import qualified Data.Set as Set
+import Data.Set qualified as Set
 import Data.Semigroup ((<>))
 import Text.Show (show)
 
 -- bv-sized
-import qualified Data.BitVector.Sized as BV
+import Data.BitVector.Sized qualified as BV
 
 -- llvm-pretty
-import qualified Text.LLVM.AST as L
+import Text.LLVM.AST qualified as L
 
 -- parameterized-utils
-import qualified Data.Parameterized.Context as Ctx
+import Data.Parameterized.Context qualified as Ctx
 import Data.Parameterized.TraversableFC (toListFC)
 
 -- crucible
-import qualified Lang.Crucible.Backend as C
-import qualified Lang.Crucible.CFG.Core as C
-import qualified Lang.Crucible.Simulator as C
+import Lang.Crucible.Backend qualified as C
+import Lang.Crucible.CFG.Core qualified as C
+import Lang.Crucible.Simulator qualified as C
 
 -- crucible-llvm
+<<<<<<< HEAD
```

Comment:
I think this was unintentionally checked into the repo. (Similarly for other merge conflict-related lines below.)

Reply:
My bad. I wonder how my compiler pass it through...